### PR TITLE
Update ruby in `.tool-versions`

### PR DESCRIPTION
**Pull request by josemigallas**

### CODE REVIEW

File: .tool-versions.sample [hunk 1]
```diff
@@ -1,3 +1,3 @@
-ruby 2.6.10
+ruby 2.7.6
 nodejs 14.21.3
 python 2.7.18
```
Comment (jlledom) [inline .tool-versions.sample:3]: Do we use python?

**Review by jlledom — APPROVED** [review]

File: .tool-versions.sample (hunk 1 above)
Comment (mayorova) [reply .tool-versions.sample:3]: I think this is something for macOS...

Comment (akostadinov) [reply .tool-versions.sample:3]: Yeah, no sensible people should be concerned...

**Review by akostadinov — APPROVED** [review]

File: .tool-versions.sample (hunk 1 above)
Comment (josemigallas) [reply .tool-versions.sample:3]: It could be commented or removed if it's a mac-only thing

**Pull request merged**